neard/build.rs

```diff
@@ -77,5 +80,8 @@ fn try_main() -> Result<()> {
 
     println!("cargo:rustc-env=NEARD_BUILD={}", get_git_version()?);
 
+    println!("cargo:rerun-if-env-changed=RUSTC");
```

Comment:
This I think isn't necessary -- Cargo already has the logic to re-run the build if anything about the compiler changes. Though, this doesn't hurt either.